sample-app-cicd/working-pipeline-secret.yaml

```diff
@@ -0,0 +1,58 @@
YjJJelFXbE1RMHB5WkZkS2JHTnROV3hrUjFaNlRHMXNka3d6VG14amJscHdXVEpXYUZreVRuWmtWelV3VEROT2JGa3pTbXhrUXpWMVdWY3hiRWxxYjJsalIyeDNXbGQ0Y0dKdFZYUmtSemx5V2xjMGRHRkhaelJPUkdOcFRFTktjbVJYU214amJUVnNaRWRXZWt4dGJIWk1NMDVzWTI1YWNGa3lWbWhaTWs1MlpGYzFNRXd6VG14amJscHdXVEpWZEZsWFRtcGlNMVoxWkVNMWRWbFhNV3hKYW05cFkwZHNkMXBYZUhCaWJWVnBURU5LY21SWFNteGpiVFZzWkVkV2VreHRiSFpNTTA1c1kyNWFjRmt5Vm1oWk1rNTJaRmMxTUV3elRteGpibHB3V1RKVmRGbFhUbXBpTTFaMVpFTTFNV0ZYVVdsUGFVa3lUbFJKTkZwVVRUQmFVekF5VGtSV2FFeFVVWHBaTWxGMFQxUlplVTFwTUhsWmFscHRUbFJDYTFscVJURk9SMDFwVEVOS2VtUlhTV2xQYVVwNlpWaE9NRnBYTURaak1sWjVaRzFzYWxwWFJtcFpNamt4WW01Uk5sb3liREJpTTBKNlRGaGtkbU50ZEhwaFJ6bDNUMjVDY0dOSFZuTmhWelZzU1c0d0xuWldVVXd5UlcxWGMyNU1jRVpJWWsxclVtVm9ha3BNVmpKMlN6SkVXbE50WVRaeFNuWkVSVEpSUjJSV1drMVVXRk5uVkV4YVdFOXVXbmd0WjNKQ04zTTJkSFJJUWxsaGJ6QmxWV05JTTAxb1dtczFUSFJoWlRaUFRtTlBjSEJHWlhSMGJUUlJWVGQ2UmxCWlRsZFdNSEZ5VjNaV1UwcERPRjloYjFBME5tTnVXWGxPTm14elkxZE5UblJ2U1d4VVVqazFNSEF3Y0haNFozWXdNalZzVG5ReFgwdFpUbkpmWldwVVFrYzRWRFJvYjNrd1ZXbFBWVTVzTUZaUVZtdHBRVFl6YldJMGVrVmpOSGQzZWpCUmNWaFhXak5ZYzA1WGVVbDNjRzUzYWtwTFFteFNUbEZ0WkhCVVdUVk5MV0ZCT1hsWmNuUmFjR2RPVUVkMWFVaG9YM0l3WTBWQ1JGWkdXVlZRYlU0NFNXaGxXVTgyYlRJNVIxQkRiRUV0ZDFOVFgxOUpTbVp1WVV0dkxUWlJRMjVRVm1kWGNVZzRXV1Z4YkZSNGRWOUlPREExY2tweWRuZFlXbmRoY2w5UVRITnFiVlJ3V1VvNVFRPT0iCiAgICB9LAogICAgInJlZ2lzdHJ5LnJlZGhhdC5pbyI6ewogICAgICAiYXV0aCI6Ik1URXdNRGt4TUROOFoybDBiM0J6TFhkdmNtdHphRzl3TFhOaE9tVjVTbWhpUjJOcFQybEtVMVY2VlhoTmFVbzVMbVY1U25wa1YwbHBUMmxLYWxsNldYbE5SMXBvVFdwYWFrNXFTVEJPUjFacldWZEpkMDFVVVROYVJFMHlXVzFSTlZwSFNYZGFRMG81TGxRelJsTkhkbFpLWWtwVGJuQjNXbVpZWHpSMmMySXpNMWRxU1dZNU5qQlBTMnd5UTA5QlNHMVVPVUpzVldaNWRERTJkbXh0ZFdsWmFYQkhSMlZhT1UxSlQybDZYM280VVdvM2NFZzRSMHBUVWxkclNpMUdNblZSU1c5NE9VTmtNRVkwTVVreVNtUnVVbXMzY1hWRE1taDRZazlyTkRoV2JGRnViMGxRYXpSUWRUbDRORGxXUTJaZmJ6QjZiVWx4TFdKTk0yWlBjMmRWYmtSUmIzZ3laMjkxZVRKVFkzTkhUWEZYVEhwNlUyaHhTM2Q2TFZCcVlYSlpTVmx1YUZJMVJEbHRiRVJFWmw5UVpVSmFSR3hPYXpKTFFscFZRM0ZWT1hGQmQxRlpMWFpUV0ZCNFFtTnZUemxmV0ZSaFkxUlFSRWczUTBWa2VsWnVTbUZFTmt4WWEySlpaa3BqTkVGQ05EbFVVbmR4TmxoYVNuWmlhbHBOVW1Zek9ETnFSVVpDVXpGVGFrZGZZM0JUZW05RVowUm9NVjlPUW1ZeFgxRk9hVVZLYkV4V1EycFRZV3hmYVZJeFowSklhbll4ZWtkMFUwTlVhVnBFZFd4amVFZHdVSFkyWkRBMlR6QXljR0paWjBwWmFXaFRXRzlEWVRJdGVHWkhjVmxxYURWVlpXWjJTWGRuTVZGVWNWOHlRV2xVYVVwclMxQjFNMFpNU2xKTWNraGpNVFJuTkc1RVQyWjRTSGd0T1Zab01qUk9PVTk0YUdOcGRYYzBWbGxJYzE4eWJqTmpNblp4T0RCMVdHcFlja1JoVWpSa2VtVlJhblo2TkZkMFZXZE1ZalF5ZGpaUGF5MXhXRXAyUkVKdVJGUnJOVE5hVjNvMlRUUnRhV2RXZEVGbFZtVk5RMXBsY0MwNE4zTnphekUwZWxWU2R6QkJiVjg0ZGs4d1ptTlNVSEZ6YUc1aFNFY3hNQzFDY2xwbE4zSk1TVTkxZDFoNVNIWmpWQzFCVXkxWE0yZGpjV1JTY1hFdFoySjJWMDB0T1Zaak5XZEtNM0o1TUY4NVRWbExlWFZRY3psNFJXSndkV0pWZDFwTmVsOUlabnBIUzJzM1ZFWk1iMVZPVVhSaE1uZG9TMmhRVm5SdVRWZExOV1o1T0U5dFFrdEZRa1ZGTkROSVNISTBhVzl5VTFOV2MzVXlRWGhVV1RSM1JqbHRjREZ2IgogICAgfSwKCSJxdWF5LmlvIjp7CiAgICAgICJhdXRoIjoiYzNOallXeHBibWNyWjJsMGIzQnpYM2R2Y210emFHOXdYM0p2WW05ME9sbFdSRkZCVkUxSk1rZFFSVlpETmt4WldsUTNURVJNV2pneVRsRkZXVFV6VjBKS1FUSXhSbGd6UmpaU1RGWllSMUJGT0VOQlYxZzBWVTA0UmxWVVRFUT0iLAogICAgICAiZW1haWwiOiIiCiAgICB9Cn0K
```

Comment:
This is the updated dockercfg that includes the auth data for the quay.io robot account:
https://quay.io/user/sscaling?tab=robots

Reply:
We will need to automate this being added to the pipeline SA dockercfg secret...